deploy/olm-catalog/ibm-block-csi-operator-community/1.7.0/ibm-block-csi-operator.v1.7.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,509 @@
+              verbs:
+              - get
+              - list
+              - watch
+              - update
+              - patch
+            - apiGroups:
+              - storage.k8s.io
+              resources:
+              - volumeattachments/status
+              verbs:
+              - patch
+            - apiGroups:
+              - storage.k8s.io
+              resources:
+              - storageclasses
+              verbs:
+              - get
+              - list
+              - watch
+            - apiGroups:
+              - monitoring.coreos.com
+              resources:
+              - servicemonitors
+              verbs:
+              - get
+              - create
+            - apiGroups:
+              - apps
+              resourceNames:
+              - ibm-block-csi-operator
+              resources:
+              - deployments/finalizers
+              verbs:
+              - update
+            - apiGroups:
+              - storage.k8s.io
+              resources:
+              - csidrivers
+              verbs:
+              - create
+              - delete
+              - get
+              - watch
+              - list
+            - apiGroups:
+              - storage.k8s.io
+              resources:
+              - csinodes
+              verbs:
+              - get
+              - list
+              - watch
+            - apiGroups:
+              - security.openshift.io
+              resourceNames:
+              - anyuid
+              - privileged
+              resources:
+              - securitycontextconstraints
+              verbs:
+              - use
+            - apiGroups:
+              - apiextensions.k8s.io
+              resources:
+              - customresourcedefinitions
+              verbs:
+              - create
+              - list
+              - watch
+              - delete
+            - apiGroups:
+              - csi.ibm.com
+              resources:
+              - '*'
+              verbs:
+              - '*'
+            - apiGroups:
+              - snapshot.storage.k8s.io
+              resources:
+              - volumesnapshotclasses
+              verbs:
+              - get
+              - watch
+              - list
+            - apiGroups:
+              - snapshot.storage.k8s.io
+              resources:
+              - volumesnapshotcontents
+              verbs:
+              - get
+              - watch
+              - list
+              - create
+              - update
+              - delete
+            - apiGroups:
+              - snapshot.storage.k8s.io
+              resources:
+              - volumesnapshotcontents/status
+              verbs:
+              - update
+            - apiGroups:
+              - snapshot.storage.k8s.io
+              resources:
+              - volumesnapshots
+              verbs:
+              - get
+              - watch
+              - list
+              - update
+              
```

Comment:
I couldn't help but notice you added spaces to this empty line (comparing to the previous csv yaml) - do we want to keep them?

Reply:
You're correct. They came from the non-community [yaml file](https://github.com/IBM/ibm-block-csi-operator/blob/3c6d21d87f618231937ae63bbd243ae694e0dbbc/deploy/olm-catalog/ibm-block-csi-operator/1.6.0/manifests/ibm-block-csi-operator.v1.6.0.clusterserviceversion.yaml#L612).
Do we need an empty line?

Reply:
why should non-community content be copied to the community one?
there is non-community content that we might not want to accidently be added to the community version.
we don't need an empty line, I'm just not sure why it should be different.
do you want them to be aligned?

Reply:
I copied the content from v1.6.0 before the last change. Hence, we have some differentiation.
I copied the relevant files again.